Dockerfile

```diff
@@ -10,5 +10,5 @@ RUN npm install
 COPY ./mapper/drop_design_docs.js /river/drop_design_docs.js
 
 ENTRYPOINT ["npm", "run"]
-CMD ["sync", "--", "--mapper=drop_design_docs", "--end_on_catchup=true"]
-#CMD ["load", "--", "--mapper=drop_design_docs"]
+CMD ["sync", "--", "--mapper=drop_design_docs.js", "--end_on_catchup=true"]
```

Comment:
Did the missing `.js` extension break anything for you? I'm just curious, and in the case that the change is necessary, I would suggest to update the README.md, accordingly.